packages/server/lib/cloud/api.ts

```diff
@@ -65,7 +64,7 @@ const rp = request.defaults((params: CypressRequestOptions, callback) => {
     rejectUnauthorized: true,
   })
 
-  const headers = params.headers != null ? params.headers : (params.headers = {})
+  const headers = params.headers ??= {}
```

Comment:
I doubled checked this, the nullish operator was introduced in Node 14, which is our minimum version. So this change should be fine 👍🏻

Reply:
because this is a `.ts` file i actually think typescript transpiles this into JS irrespective of the node version. but also good to know node v14 supports all